### PR TITLE
Clean dependencies and fix tests

### PR DESCRIPTION
**Pull request by orblazer**
Hello,
I have clean unused dependencies and depcreated.

The "refactor: Change WS library by UWS" wrong here, the PR #1124 need merged before.

Thanks.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -82,19 +78,15 @@
...
     "ieee754": "^1.1.8",
-    "inquirer": "^3.0.6",
     "lego-api": "^1.0.7",
     "mustache": "^2.3.0",
     "postcss": "^6.0.1",
     "pretty-time": "^0.2.0",
     "prettysize": "0.0.3",
     "realm-utils": "^1.0.9",
-    "regexpu-core": "^4.1.3",
-    "request": "^2.79.0",
```
Comment (nchanged) [inline package.json]: We need request, Quantum uses it with one option

File: src/devServer/SocketServer.ts [hunk 2]
```diff
@@ -1,5 +1,5 @@
 import { FuseBox } from "../core/FuseBox";
-import { Server } from "ws";
```
Comment (nchanged) [inline src/devServer/SocketServer.ts]: I think that belongs to the different PR ;-)

File: package.json [hunk 3]
```diff
@@ -82,19 +78,15 @@
     "fuse-concat-with-sourcemaps": "^1.0.5",
     "glob": "^7.1.1",
     "ieee754": "^1.1.8",
-    "inquirer": "^3.0.6",
```
Comment (nchanged) [inline package.json]: Are you sure it's not in use anywhere?

File: src/devServer/SocketServer.ts (hunk 2 above)
Comment (orblazer) [reply src/devServer/SocketServer.ts]: yep but i dont know how to skip this commit :(

File: package.json (hunk 3 above)
Comment (orblazer) [reply package.json]: Yep, i have search in all files on github and in the project

File: package.json (hunk 1 above)
Comment (orblazer) [reply package.json]: Hump i don't find the usage in another place than test

Comment (nchanged) [reply package.json]: https://github.com/fuse-box/fuse-box/blob/8aa160b142e6a3d069b192aa19b271e47e329d2a/modules/fuse-box-responsive-api/index.js#L187-L194

File: src/devServer/SocketServer.ts (hunk 2 above)
Comment (nchanged) [reply src/devServer/SocketServer.ts]: I will check your first PR soon, then you can rebase

File: package.json (hunk 3 above)
Comment (nchanged) [reply package.json]: 👍

File: package.json (hunk 1 above)
Comment (orblazer) [reply package.json]: ops, ok i rewrite commit history for fix that

**Comment by orblazer** [comment]
Ok, i have rewrite the history of commits for cleanup.
Normally this PR is OK.

**Comment by orblazer** [comment]
Hum... i have an problem, the test "Should replace an alias - as a file - with babel" not pass because the alias is not apply.

For begin fix i have replace that condition [ImportDeclaration.ts#L40](https://github.com/fuse-box/fuse-box/blob/master/src/analysis/plugins/ImportDeclaration.ts#L40) by :
```ts
if (node.source && analysis.nodeIsString(node.source)) {
    let requireStatement = this.handleAliasReplacement(file, node.source.value);
    if (requireStatement) {
        analysis.registerReplacement(node.source.value, requireStatement);
        node.source.value = requireStatement;
        analysis.addDependency(requireStatement);
    }
}
```
But now, this line in [FileAnalysis.ts#L150](https://github.com/fuse-box/fuse-box/blob/master/src/analysis/FileAnalysis.ts#L150) not apply the alias because the content of file is :
```js
export { default as canada } from 'eh';
```
We probably need change the regex but i don't know where to replace her.

**Comment by orblazer** [comment]
Hum i don't know, probably because i have update `babel-preset-latest` to `babel-preset-env` and the output code is ES2015. Probably before that the output code is ES5 and not ES6.

**Comment by nchanged** [comment]
@orblazer can you cleanup this PR?

**Comment by orblazer** [comment]
Yes, but i close this PR and make new clean PR.
